google-codelab-step/google_codelab_step.js

```diff
@@ -133,11 +133,16 @@ class CodelabStep extends HTMLElement {
     this.inner_.appendChild(title);
 
     const children = Array.from(dom.getChildren(this));
-    console.log(children);
     children.forEach((c) => {
       dom.appendChild(this.inner_, (c));
     });
 
+    const codeElements = this.inner_.querySelectorAll('code');
+    codeElements.forEach((el) => {
+      const code = window['prettyPrintOne'](el.innerHTML);
```

Comment:
forgot to mention: maybe it's better to use `el.textContent`? code should be plain text anyway.

Reply:
Done